### PR TITLE
Improve autocomplete

### PR DESCRIPTION
**Pull request by eoinkelly**
Sort autocomplete results and increase limit

This change fixes issue #490. It:

* Increases the limit on autocomplete results from 10 to 100.
* Sort autocomplete results alphabetically
* Stop logging about getting too many results from Freelex because we don't care anymore.

### CODE REVIEW

**Comment by eoinkelly** [comment]
@DSRU I have made the requested changes. I kept a limit because it will help protect the user from getting a big wad of text as autocomplete if there is ever a bug between us and Freelex. I have used it locally and it seems to work well (the sorting is a good improvement I think) but you can have a play when it hits staging and see if it does what you expect.

File: app/services/autocomplete_search_service.rb [hunk 1]
```diff
@@ -45,20 +45,12 @@ def find_suggestions # rubocop:disable Metrics/MethodLength, Metrics/AbcSize
...
-          #{results}
-      EO_MSG
-
-      @logger.info(msg)
-    end
-
-    results.take(MAX_NUM_SUGGESTIONS)
+    response
+      .body
+      .force_encoding(Encoding::UTF_8)
```
Comment (Br3nda) [inline app/services/autocomplete_search_service.rb:50]: Do we still need this? Had it been fixed on freelex?

**Review by Br3nda — APPROVED** [review]

File: app/services/autocomplete_search_service.rb (hunk 1 above)
Comment (eoinkelly) [reply app/services/autocomplete_search_service.rb:50]: I haven't heard anything from Dave yet so I assume not. I'll ask again just in case.

**Comment by DSRU** [comment]
Dave hasn't been able to get in to the university servers yet - he's been trying to sort it out with them.

________________________________

**Pull request merged**